app/src/components/Chat/Extensions/SwearFilter.jsx

```diff
@@ -0,0 +1,3 @@
+export default (renderingItem) => {
+    renderingItem.data.text = renderingItem.data.text.replace("crap", "****");
```

Comment:
This will eventually be replaced with a content moderation service?

Reply:
haha, yeah <3